### PR TITLE
Fixes for multiple pretrained model files

### PR DESCRIPTION
**Pull request by lukeyeager**
Close https://github.com/NVIDIA/DIGITS/issues/849.

See explanation at https://github.com/NVIDIA/DIGITS/pull/498#issuecomment-226550671.

### CODE REVIEW

**Comment by lukeyeager** [comment]
@gheinrich can you take a look?

File: digits/model/tasks/torch_train.py [hunk 1]
```diff
@@ -250,7 +250,10 @@ def task_arguments(self, resources, env):
             args.append('--type=float')
 
         if self.pretrained_model:
-            args.append('--weights=%s' % self.path(self.pretrained_model))
+            filenames = self.pretrained_model.split(os.path.pathsep)
+            if len(filenames) > 1:
+                raise ValueError('Torch does not support multiple pretrained model files')
+            args.append('--weights=%s' % self.path(filenames[0]))
```
Comment (gheinrich) [inline digits/model/tasks/torch_train.py]: that looks good though (probably not due to this) it seems the error came too quickly to show in the UI, as if we hadn't yet created the socketIO interface. The error message only showed after I reloaded the page.
I tested that it works when only one pretrained model is provided.

**Comment by gheinrich** [comment]
This looks good to me. On Linux this is working for me. It's difficult for me to test on Windows as I don't have the set-up. I just verified that `os.pathsep` returns `;` on Windows.

**Comment by IsaacYangSLA** [comment]
I will test it here on Windows.

**Comment by IsaacYangSLA** [comment]
Did the new image classification model page remove pre-trained weights field?  I couldn't find it on 4.0.0-rc.2.

**Comment by lukeyeager** [comment]
> Did the new image classification model page remove pre-trained weights field?

Look under the "Custom Network" tab

**Comment by IsaacYangSLA** [comment]
Thanks.  My bad.  Still a lot to learn about DIGITS.
I tested on Windows.  Separating multiple weight files with ';' and path starts with /, both caffe.exe and pycaffe can load those weights.  However, there is a bug in _pycaffe_args, so I need to send a PR on loading_script portion of _pycaffe_args.

**Comment by lukeyeager** [comment]
Great, so we're good? Please merge if so.

**Comment by IsaacYangSLA** [comment]
Looks great.  I believe it works good in Windows now.

**Pull request merged**